js/binance.js

```diff
@@ -4141,18 +4147,55 @@ module.exports = class binance extends Exchange {
         await this.loadMarkets ();
         const currency = this.currency (code);
         let type = this.safeString (params, 'type');
-        if (type === undefined) {
-            const accountsByType = this.safeValue (this.options, 'accountsByType', {});
-            const fromId = this.safeString (accountsByType, fromAccount, fromAccount);
-            const toId = this.safeString (accountsByType, toAccount, toAccount);
-            type = fromId + '_' + toId;
-        }
         const request = {
             'asset': currency['id'],
             'amount': this.currencyToPrecision (code, amount),
-            'type': type,
         };
-        const response = await this.sapiPostAssetTransfer (this.extend (request, params));
+        let method = 'sapiPostAssetTransfer';
+        if (type === undefined) {
+            let fromId = this.typeToAccount (fromAccount);
+            let toId = this.typeToAccount (toAccount);
+            const fromIsolated = this.ids.includes (fromId);
```

Comment:
we can't use  `includes( )` because it's not transpiled correctly.  Consider using something like `this.inArray`